tests/features/java/openjdk.feature

```diff
@@ -34,8 +34,18 @@ Feature: Miscellaneous OpenJDK-related unit tests
     Then available container log should not contain java-1.8.0
     Then available container log should not contain java-11
 
-  @ubi8
-  @openjdk
+  @ubi8/openjdk-8
+  @ubi8/openjdk-8-runtime
+  Scenario: Ensure JAVA_HOME is defined and contains Java
+    When container is started with args
+    | arg     | value                                  |
+    | command | bash -c "$JAVA_HOME/jre/bin/java -version" |
```

Comment:
This would not pass for any of the full builder images, since $JAVA_HOME/jre does not exist in those.

Reply:
The correct test already exists here:
https://github.com/jboss-container-images/openjdk/blob/develop/tests/features/java/openjdk.feature#L37 
this should continue to pass without being altered.